dio/README.md

```diff
@@ -0,0 +1,39 @@
+# dio
+
+diamond-io CLI implementation
+
+### install
+```
+cargo install --path .
+```
+disk feature enable
+```
+cargo install --features disk --path .
+```
+debug feature enable
+```
+cargo install --features debug --path .
+```
+
+### Run
+
+Execute the end-to-end workflow from obfuscation to evaluation:
+
+```
+dio run-bench \
+  -c {CONFIG-TOML-PATH} \
+  -o {OBFUSCATION-DIRECTORY-PATH} \
+  --add-num {ADD-GATE-NUMBER} \
+  --mul-num {MUL-GATE-NUMBER}
+```
+
+### Build Circuit
```

Comment:
Do we really need a separate functionality to count the the circuit gates? Can't we add it as an optional (or even default) feature to the `run-bench` command? I understand that you implemented to collect this info because we already run the benches before, but I expect that by default all the information can be collected in a single process